### PR TITLE
Feat/native/ts5 correct images

### LINKED ISSUE #12800: Device icon fixup
Please change the icon in wallet switcher based on [this message](https://satoshilabs.slack.com/archives/C04JZAYEV41/p1718006965804489). SVG attached in there as well.

### PR DESCRIPTION
**Pull request by vytick**
Updated images for TS5 to sync with release

Resolve [#12800](https://github.com/trezor/trezor-suite/issues/12800)

### CODE REVIEW

File: suite-common/icons/assets/icons/trezorT3T1.svg
Comment (matejkriz) [inline suite-common/icons/assets/icons/trezorT3T1.svg:1]: I think this is not the correct icon.

**Comment by PeKne** [comment]
If you are already changing the device icon, please close this issue after merge: https://github.com/orgs/trezor/projects/61/views/16?filterQuery=icon&pane=issue&itemId=66750042 🙏

File: suite-common/icons/assets/icons/trezorT3T1.svg
Comment (PeKne) [reply suite-common/icons/assets/icons/trezorT3T1.svg:1]: Here is related issue https://github.com/orgs/trezor/projects/61/views/16?filterQuery=icon&pane=issue&itemId=66750042 🙏

Comment (vytick) [reply suite-common/icons/assets/icons/trezorT3T1.svg:1]: 2d339b1713

Comment (vytick) [reply suite-common/icons/assets/icons/trezorT3T1.svg:1]: Screenshot 2024-06-17 at 12 17 25

**Review by matejkriz — APPROVED** [review]

**Pull request merged**